src/NumericalAlgorithms/Interpolation/InterpolationTargetWedgeSectionTorus.hpp

```diff
@@ -0,0 +1,220 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <pup.h>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "NumericalAlgorithms/Interpolation/SendPointsToInterpolator.hpp"
+#include "NumericalAlgorithms/Interpolation/Tags.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace intrp {
+
+namespace OptionHolders {
```

Comment:
This namespace should be documented so it shows up and we can find `WedgeSectionTorus`. (That may not be your fault, I just checked the docs and noticed)